Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2.yml

```diff
@@ -0,0 +1,2250 @@
+      description: Application value.
+      isArray: false
+      name: value
+      required: true
+      secret: false
+    - default: false
+      description: The policy ID.
+      isArray: false
+      name: policyId
+      required: true
+      secret: false
+    deprecated: false
+    description: 'Updates an existing rule with a new rule. Note: System
+      policies cannot be modified.'
+    execution: false
+    name: cbd-update-rule-in-policy
+  - arguments:
+    - default: false
+      description: The policy ID.
+      isArray: false
+      name: policyId
+      required: true
+      secret: false
+    - default: false
+      description: The rule ID.
+      isArray: false
+      name: ruleId
+      required: true
+      secret: false
+    deprecated: false
+    description: 'Removes a rule from an existing policy. Note: System policies
+      cannot be modified.'
+    execution: false
+    name: cbd-delete-rule-from-policy
+  - arguments:
+    - default: false
+      description: The job ID.
+      isArray: false
+      name: job_id
+      required: true
+      secret: false
+    - default: false
+      description: The number of rows to request. Can be paginated. Default is 10.
+      isArray: false
+      name: rows
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieves the result for an enriched events search request for a
+      given job ID. By default returns 10 rows.
+    execution: false
+    name: cbd-find-events-results
+    outputs:
+    - contextPath: CarbonBlackDefense.Events.Results
+      description: The results of the event.
+      type: Unknown
+    - contextPath: CarbonBlackDefense.Events.Results.approximate_unaggregated
+      description: The approximate number of unaggregated results.
+      type: Number
+    - contextPath: CarbonBlackDefense.Events.Results.completed
+      description: The number of completed results.
+      type: Number
+    - contextPath: CarbonBlackDefense.Events.Results.contacted
+      description: The number of contacted results.
+      type: Number
+    - contextPath: CarbonBlackDefense.Events.Results.num_aggregated
+      description: The number of aggregated results.
+      type: Number
+    - contextPath: CarbonBlackDefense.Events.Results.num_available
+      description: The number of events available in this search.
+      type: Number
+    - contextPath: CarbonBlackDefense.Events.Results.num_found
+      description: The number of events found in this search.
+      type: Number
+    - contextPath: CarbonBlackDefense.Events.Results.results
+      description: The lists that contains the data of the results for this search.
+      type: Unknown
+  - arguments:
+    - default: false
+      description: A comma-separated list of event IDs to fetch. 
+      isArray: false
+      name: event_ids
+      required: true
+      secret: false
+    deprecated: false
+    description: Initiates a request to retrieve detail fields for enriched events.
```

Comment:
I would add here that this job ID can be used to get the results using the "cbd-find-events-details-results" command.